README.md

```diff
@@ -5,167 +5,142 @@
 [![Build Status][1]][2]
 
 ## Overview
-Heptio Ark is a utility for managing disaster recovery, specifically for your [Kubernetes][14] cluster resources and persistent volumes. It provides a simple, configurable, and operationally robust way to back up and restore applications and PVs from a series of checkpoints. This allows you to better automate in the following scenarios:
 
-* **Disaster recovery** with reduced TTR (time to respond), in the case of:
-    * Infrastructure loss
-    * Data corruption
-    * Service outages
+Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. Ark lets you:
 
-* **Cross-cloud-provider migration** for Kubernetes API objects (cross-cloud-provider migration of persistent volume snapshots not yet supported)
+* Manage disaster recovery.
+* Copy cluster resources across cloud providers. NOTE: PersistenVolume objects are not supported.
```

Comment:
Typo: PersistentVolume

This almost makes it sound like Ark doesn't support PVs at all, when in fact we do support them, just not for cross-cloud-provider restores.